src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -18,13 +19,19 @@ export default class SimidMapCreative extends BaseSimidCreative {
 
   constructor() {
     super();
+    /**
+     * An instance of a user session
+     * @private {?UserActivityLogger}
```

Comment:
` @private @const {!UserActivityLogger}`
`@const` because its not changing, `!` instead of `?` because it will never be null

Reply:
fixed